app/models/pipeline_run_stage.rb

```diff
@@ -310,6 +310,35 @@ def db_load_postprocess
       mysqlimport --replace --local --user=$DB_USERNAME --host=#{rds_host} --password=$DB_PASSWORD --columns=taxid,hit_type,first_byte,last_byte,pipeline_run_id --fields-terminated-by=',' idseq_#{Rails.env} taxon_byteranges;
     `
     _stdout, _stderr, _status = Open3.capture3("rm -f #{downloaded_byteranges_path}")
+    notify_users if notify?
+  end
+
+  def project_id
+    pipeline_run.sample.project_id
+  end
+
+  def notify?
+    statuses_in_project = PipelineRun.connection.select_all("
+      SELECT
+        pipeline_runs.job_status
```

Comment:
this is problematic when there are other previous runs that failed. 

This will be my query 

x = PipelineRun.where("id in (select max(id) from pipeline_runs group by sample_id) and sample_id in (select id from samples where project_id = #{project_id.to_i})").where("status != ?",   PipelineRun::STATUS_CHECKED)
x.count > 0

Reply:
thanks!!